src/tactic/core.lean

```diff
@@ -842,6 +842,7 @@ do
   return r
 
 namespace interactive
+/-- Run a tactic "under binders", by running `intros` before, and `revert` afterwards. -/
 meta def under_binders (i : itactic) : itactic := tactic.under_binders i
 end interactive
```

Comment:
```suggestion
run_cmd add_interactive [``under_binders]
```

Reply:
I tried this, actually, but it didn't give me a working interactive tactic. The passed tactic block isn't accepted.

Reply:
Oh right, it literally copies the declaration, with the same types.  If you used `itactic` for the argument in the `tactic.under_binders` version, then it should work.

Reply:
But there I want to allow the possibility of a tactic that returns something, for non-interactive use.

Reply:
Ok, got it.